### DISCUSSION TITLE
How to get counts?

### DISCUSSION THREAD

**Discussion by rohit-chhabra** [shihabmridha/nodejs-repository-pattern-and-ioc#11] [Q&A] [answered]
Good pattern... really appreciate sharing this... How do you recommend finding the count of collections in a repository. It seems that would be needed to find how many total pages would be needed on the UI to show page numbers. Do you recommend a generic method in the repository base class to support it?

**Comment by shihabmridha** [comment, marked as answer]
Yes, I would put it in the base repository class.
